yt_dlp/options.py

```diff
@@ -1400,12 +1400,14 @@ def _alias_callback(option, opt_str, value, parser, opts, nargs):
         help='Do not read/dump cookies from/to file (default)')
     filesystem.add_option(
         '--cookies-from-browser',
-        dest='cookiesfrombrowser', metavar='BROWSER[+KEYRING][:PROFILE[:CONTAINER]]',
+        dest='cookiesfrombrowser', metavar='BROWSER[+KEYRING][:PROFILE][:CONTAINER]',
```

Comment:
in my opinion this line should be reverted, because otherwise it's not clear that 'BROWSER:CONTAINER' doesn't work (I haven't tested this PR, but I'm guessing that that wouldn't work)

Reply:
theoretically this line should be 'BROWSER[+KEYRING][:[PROFILE][:CONTAINER]]', but that might be too confusing.

PS: I am not a maintainer, these two comments are just my 5 cents on this PR

Reply:
Actually, to make it easy to document, I would suggest modifying the code so that 'CONTAINER' is always proceeded by a '::'